moveit_ros/moveit_servo/test/test_servo_status.cpp

```diff
@@ -90,6 +90,21 @@ TEST_F(ServoFixture, ReachSingular)
   }
   EXPECT_EQ(getLatestStatus(), moveit_servo::StatusCode::NO_WARNING);
 
+  // Test a joint limit
+  rclcpp::Time start_time = node_->now();
+  rclcpp::Duration loop_duration(0, 0);
+  while (loop_duration.seconds() < 5)
+  {
+    loop_duration = node_->now() - start_time;
+    auto joint_msg = std::make_unique<control_msgs::msg::JointJog>();
+    joint_msg->header.stamp = node_->now();
+    joint_msg->header.frame_id = "panda_link3";
+    joint_msg->velocities.push_back(2.0);
+    pub_joint_cmd_->publish(std::move(joint_msg));
+    publish_loop_rate.sleep();
+  }
+  EXPECT_EQ(getLatestStatus(), moveit_servo::StatusCode::JOINT_BOUND);
```

Comment:
This test seems to fail in CI where the status code that `getLatestStatus()` returns is 0 but `moveit_servo::StatusCode::JOINT_BOUND` is 5 (see https://github.com/ros-planning/moveit2/pull/451/checks?check_run_id=2506237803). Is there a specific reason why we are publishing 2.0 velocity? Maybe adding a comment about what the joint limits are would be nice.

Reply:
You're right. I'll refactor the test.

No reason to choose 2.0 for velocity other than it's a reasonable value.